assets/js/components/Table/Table.test.jsx

```diff
@@ -119,5 +119,20 @@ describe('Table component', () => {
         expect(table.querySelectorAll('tbody > tr')).toHaveLength(1);
       });
     });
+
+    it('should return empty state message when data is empty', async () => {
+      const data = [];
+      const emptyStateMessage = 'No data available';
+
+      render(
+        <Table config={tableConfig} data={data} setSearchParams={() => {}} />
```

Comment:
```suggestion
      const emptyStateMessage = 'No data available';

      render(
        <Table config={tableConfig} data={[]} setSearchParams={() => {}} />
```

Reply:
I adjusted the emptyState component.
Now it accepts a emptyStateText prop so we can have different messages and the test now proves it.